Xamarin.PropertyEditing.Mac/Controls/PredefinedValuesEditor.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
```

Comment:
Maybe edit the commit message and change `valules` -> `values` just in case we want to search the commit logs later ;p